### PR TITLE
Point to the test run instead of listing the tests

### PR DESCRIPTION
**Pull request by situmam**
The workflow is improved. Plan is to have the list of tests displayed in the test run which eliminates the need for the tests page. 

Signed-off-by: Anmar Oueja <sitummam@gmail.com>

### CODE REVIEW

**Comment by danrue** [comment]
Ok, it took me a while to figure out what this change did. Next time please be more explicit in the explanation.

This commit changes the navigation so that when you click on a failing test result in the build view, you go to the test run page (which contains the metadata and all test runs) rather than the test result page.

This isn't ideal, but it is better than the current behavior. +1.

**Comment by situmam** [comment]
Hrmm.. my comment got truncated for some reason. I will be careful next time.

File: squad/frontend/templates/squad/build.html [hunk 1]
```diff
@@ -20,10 +20,11 @@ <h2>Test results</h2>
         </form>
     </div>
 </div>
+
 <div class='test-results highlight-row'>
 {% for status in statuses %}
 {% if status.suite and status.has_tests %}
-<a href="{% testrun_suite_tests_url status %}" class="status-fail{{status.tests_fail}}" {% if status.tests_fail == 0%}style='display: none'{% endif%} id='tests-{{status.id}}' ng-show='match("tests-" + {{status.id}})'>
+<a href="{%  url "testrun" status.test_run.build.project.group.slug status.test_run.build.project.slug status.test_run.build.version status.test_run.job_id %}" class="status-fail{{status.tests_fail}}" {% if status.tests_fail == 0%}style='display: none'{% endif%} id='tests-{{status.id}}' ng-show='match("tests-" + {{status.id}})'>
```
Comment (terceiro) [inline squad/frontend/templates/squad/build.html]: you can replace all that by `project_url test_run`

Comment (situmam) [reply squad/frontend/templates/squad/build.html]: I tried but it is erroring out on the web interface with the following: `In template /app/squad/frontend/templates/squad/base.html, error at line 16` Unfortunately that doesn't tell me much but the consolr of the docker shows a lot of errors below. I am guessing it is a context issue where it isn't setup properly. 

`[2018-05-18 18:33:45 +0000] [ERROR] Internal Server Error: /lkft/linux-mainline-oe/build/v4.13-rc6-50-g98b9f8a45499/
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.5/dist-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/app/squad/http.py", line 47, in auth_wrapper
    return func(*args, **kwargs)
  File "/app/squad/frontend/views.py", line 117, in build
    return render(request, 'squad/build.html', context)
  File "/usr/local/lib/python3.5/dist-packages/django/shortcuts.py", line 30, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/usr/local/lib/python3.5/dist-packages/django/template/loader.py", line 68, in render_to_string
    return template.render(context, request)
  File "/usr/local/lib/python3.5/dist-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 207, in render
    return self._render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.5/dist-packages/django/template/library.py", line 203, in render
    output = self.func(*resolved_args, **resolved_kwargs)
  File "/app/squad/frontend/templatetags/squad.py", line 38, in project_url
    project = the_object.project
AttributeError: 'str' object has no attribute 'project'
[2018-05-18 18:33:46 +0000] [ERROR] "GET /lkft/linux-mainline-oe/build/v4.13-rc6-50-g98b9f8a45499/ HTTP/1.1" 500 238575
[2018-05-18 18:33:46 +0000] [INFO] "GET /favicon.ico HTTP/1.1" 302 0
[2018-05-18 18:33:46 +0000] [INFO] "GET /static/favicon.ico HTTP/1.1" 200 5534`

Comment (terceiro) [reply squad/frontend/templates/squad/build.html]: ah yes, I meant `project_url status.test_run`.

just `testrun` is not defined in this context

Comment (situmam) [reply squad/frontend/templates/squad/build.html]: I tested it and it works. I will push an update to this pull request.

**Pull request merged**